docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -60,44 +60,14 @@ A connection is an automated data pipeline that replicates data from a source to
 
 Setting up a connection involves configuring the following parameters:
 
-<table>
-  <tr>
-   <td><strong>Parameter</strong>
-   </td>
-   <td><strong>Description</strong>
-   </td>
-  </tr>
-  <tr>
-   <td>Replication frequency
-   </td>
-   <td>How often should the data sync?
-   </td>
-  </tr>
-  <tr>
-   <td>Destination Namespace and stream names
-   </td>
-   <td>Where should the replicated data be written?
-   </td>
-  </tr>
-  <tr>
-   <td>Catalog selection
-   </td>
-   <td>Which streams and fields should be replicated from the source to the destination?
-   </td>
-  </tr>
-  <tr>
-   <td>Sync mode
-   </td>
-   <td>How should the streams be replicated (read and written)?
-   </td>
-  </tr>
-  <tr>
-   <td>Optional transformations
-   </td>
-   <td>How should Airbyte protocol messages (raw JSON blob) data be converted into other data representations?
-   </td>
-  </tr>
-</table>
+| Parameter                              | Description                                                                                                                                                                                                                                                           |
+|----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Replication frequency                  | How often should the data sync?                                                                                                                                                                                                                                       |
+| Data residency                         | Where is the data processing location for this connection? To choose the preferred data processing location for all of your connections, set your default [data residency](https://docs.airbyte.com/cloud/managing-airbyte-cloud#choose-your-default-data-residency). |
```

Comment:
That might be a bit of a nit but I find the wording here a bit odd: "Where is the data processing location for this connection". I'm not sure I would understand what this means if I were a customer (I haven't checked our documentation for a while, so my opinion might be off.) Every other entry uses the word `should` which really conveys that those options are directive for Airbyte Cloud.

I would go with something like this instead: "Where should Airbyte Cloud data synchronization workflow run"

Reply:
I agree this sounds weird. 

Maybe something even simpler - "Where do I want Airbyte to process my data?"

Reply:
Great points! I will change the wording here